csvcubed/tests/unit/writers/test_qbwriter.py

```diff
@@ -432,6 +503,57 @@ def test_default_property_value_uris_single_measure_obs_val():
     assert default_value_uri is None
 
 
+def test_output_new_code_list_csvws_urls():
+    data = pd.DataFrame({"New Dimension": ["A", "B", "C"], "Value": [1, 2, 3]})
+    cube = Cube(
+        CatalogMetadata("Cube Name"),
+        pd.DataFrame(),
+        [
+            QbColumn(
+                "New Dimension",
+                NewQbDimension.from_data("Some Dimension", data["New Dimension"]),
+            )
+        ],
+    )
+    qb_writer = QbWriter(cube)
+    with TemporaryDirectory() as temp_dir:
+        temp_dir = Path(temp_dir)
+        qb_writer._output_new_code_list_csvws(temp_dir)
+        graph = Graph()
+        graph.parse(temp_dir / "some-dimension.csv-metadata.json")
+        assert (
+            URIRef(f"file://{temp_dir}/some-dimension.csv#code-list"),
```

Comment:
This may well cause some problems on Jenkins if the file/folder path contains chars like `@`, etc. could you use a `publicID` when you parse instead so we can be sure to side-step the problem?